src/java.desktop/share/classes/java/awt/font/TextLine.java

```diff
@@ -836,14 +836,18 @@ public Rectangle2D getVisualBounds() {
         }
 
         if (result == null) {
-            result = new Rectangle2D.Float(Float.MAX_VALUE, Float.MAX_VALUE, Float.MIN_VALUE, Float.MIN_VALUE);
```

Comment:
That's interesting. I'd have assumed what you changed it to would be  the answer in such. a case ? Why was it like this ? Perhaps it was never possible before and this was intended to 'flag a problem' ??